src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -114,266 +112,218 @@ public static List<String> createAliasesWithOid(String ... oids) {
-            add(p, "CertStore", "Collection",
-                    "sun.security.provider.certpath.CollectionCertStore",
-                    null, attrs);
-            add(p, "CertStore", "com.sun.security.IndexedCollection",
-                    "sun.security.provider.certpath.IndexedCollectionCertStore",
-                    null, attrs);
-
-            /*
-             * Policy
-             */
-            add(p, "Policy", "JavaPolicy", "sun.security.provider.PolicySpiFile",
-                    null, null);
-
-            /*
-             * Configuration
-             */
-            add(p, "Configuration", "JavaLoginConfig",
-                    "sun.security.provider.ConfigFile$Spi", null, null);
-
-            /*
-             * CertPathBuilder and CertPathValidator
-             */
-            attrs.clear();
-            attrs.put("ValidationAlgorithm", "RFC5280");
-            attrs.put("ImplementedIn", "Software");
-
-            add(p, "CertPathBuilder", "PKIX",
-                    "sun.security.provider.certpath.SunCertPathBuilder",
-                    null, attrs);
-            add(p, "CertPathValidator", "PKIX",
-                    "sun.security.provider.certpath.PKIXCertPathValidator",
+        /*
+         * SecureRandom engines
+         */
+        attrs.put("ThreadSafe", "true");
+        if (NativePRNG.isAvailable()) {
+            add(p, "SecureRandom", "NativePRNG",
+                    "sun.security.provider.NativePRNG",
                     null, attrs);
-        } else {
-            /*
-             * SecureRandom engines
-             */
-            attrs.put("ThreadSafe", "true");
-            if (NativePRNG.isAvailable()) {
-                add(p, "SecureRandom", "NativePRNG",
-                        "sun.security.provider.NativePRNG",
-                        null, attrs);
-            }
-            if (NativePRNG.Blocking.isAvailable()) {
-                add(p, "SecureRandom", "NativePRNGBlocking",
-                        "sun.security.provider.NativePRNG$Blocking", null, attrs);
-            }
-            if (NativePRNG.NonBlocking.isAvailable()) {
-                add(p, "SecureRandom", "NativePRNGNonBlocking",
-                        "sun.security.provider.NativePRNG$NonBlocking", null, attrs);
-            }
-            attrs.put("ImplementedIn", "Software");
-            add(p, "SecureRandom", "DRBG", "sun.security.provider.DRBG",
+        }
+        if (NativePRNG.Blocking.isAvailable()) {
+            add(p, "SecureRandom", "NativePRNGBlocking",
+                    "sun.security.provider.NativePRNG$Blocking", null, attrs);
+        }
+        if (NativePRNG.NonBlocking.isAvailable()) {
+            add(p, "SecureRandom", "NativePRNGNonBlocking",
+                    "sun.security.provider.NativePRNG$NonBlocking", null, attrs);
+        }
+        attrs.put("ImplementedIn", "Software");
+        add(p, "SecureRandom", "DRBG", "sun.security.provider.DRBG",
                 null, attrs);
```

Comment:
@keithc-ca 

For these example lines, I compared and did not see they are different than in https://github.com/ibmruntimes/openj9-openjdk-jdk11/commit/d9adaafe153c9d6e5826d1f7fa72e1e38c20e20c. For the SunEntries.java file, in the current commit, I removed all the FIPS checks made by commit https://github.com/ibmruntimes/openj9-openjdk-jdk11/commit/eaced24d8ce3b97bad578e11689f826c7ef9412b, because we don't need to check FIPS in SunEntries.java anymore. 

Could you help to provider some detailed examples about what changes need to revert to make them same as commit https://github.com/ibmruntimes/openj9-openjdk-jdk11/commit/d9adaafe153c9d6e5826d1f7fa72e1e38c20e20c ?